samples/cxl_mem_tg/cxl_tg_test.h

```diff
@@ -84,7 +84,7 @@ class cxl_tg_test : public test_command {
 
   // Convert number of transactions to bandwidth (GB/s)
   double bw_calc(uint64_t xfer_bytes, uint64_t num_ticks) {
-    return (double)(xfer_bytes) /
+    return (double)(xfer_bytes)* 0.931323 /
```

Comment:
Use a \#define instead of the magic number.

Reply:
Fixed added  #define